plugin/src/test/groovy/com/github/triplet/gradle/play/PlayPublishTaskTest.groovy

```diff
@@ -107,57 +107,6 @@ class PlayPublishTaskTest {
         doReturn(editCommitMock).when(editsMock).commit(anyString(), anyString())
     }
 
-    @Test
-    void testApplicationId() {
-        def project = TestHelper.evaluatableProject()
-        project.evaluate()
-
-        // Attach the mock
-        setMockPublisher(project.tasks.publishApkRelease)
-
-        // finally run the task we want to check
-        project.tasks.publishApkRelease.publishApks()
-
-        // verify that we init the connection with the correct application id
-        verify(editsMock).insert('com.example.publisher', null)
-    }
-
-    @Test
-    void testApplicationIdWithFlavorsAndSuffix() {
-        def project = TestHelper.evaluatableProject()
-
-        project.android {
-            flavorDimensions 'pricing'
-
-            productFlavors {
-                paid {
-                    dimension 'pricing'
-                    applicationId 'com.example.publisher.paid'
-                }
-                free {
-                    dimension 'pricing'
-                }
-            }
-
-            buildTypes {
-                release {
-                    applicationIdSuffix '.release'
-                }
-            }
-        }
-
-        project.evaluate()
-
-        // Attach the mock
-        setMockPublisher(project.tasks.publishApkPaidRelease)
-
-        // finally run the task we want to check
-        project.tasks.publishApkPaidRelease.publishApks()
-
-        // verify that we init the connection with the correct application id
-        verify(editsMock).insert('com.example.publisher.paid.release', null)
-    }
-
```

Comment:
Why did we have to remove those tests? Those tests shouldn't be broken because they make sure we upload to the correct application ID

Reply:
Yeah, they broke because there isn't an easy way to call the @TaskAction method with fake inputs. In the end, I decided to remove the tests because they're literally just checking one line:
https://github.com/Triple-T/gradle-play-publisher/blob/239d3aae4d86a66fd0f9b400589d8090e04347d1/plugin/src/main/kotlin/com/github/triplet/gradle/play/internal/PlayPublishTaskBase.kt#L52

And it's not even specific to the APK task. If you want, we could add a test for that but I don't really see the point in check a simple line of code. 🤷‍♂️

Reply:
Those tests were actually quite useful at the time. In #54, #55 we found out that reading the application id from the variant at runtime makes sure we pick up any changes that happen after evaluation (which we didn't support before).

Completely removing them is a bit scary. Plus, we definitely need a way to call task actions in tests even with the new input parameters. We should at least keep one of them or a simplified version (preferably the one about changing the application id after evaluate). That would also serve as a template of how to mock task inputs

Reply:
Found an easy way to bring back that test. I'll add a commit if you don't mind